target_bigquery/target.py

```diff
@@ -590,4 +591,9 @@ def _validate_config(
         self, raise_errors: bool = True, warnings_as_errors: bool = False
     ) -> Tuple[List[str], List[str]]:
         """Don't throw on config validation since our JSON schema doesn't seem to play well with meltano for whatever reason"""
-        return super()._validate_config(False, False)
+        return super()._validate_config(raise_errors=False)
+
+    # orjson does not support decimal.Decimal, so just override deserialization
+    # behaviour to use orjson directly
+    def deserialize_json(self, line):
+        return orjson.loads(line)
```

Comment:
Be aware that this override will no longer be called for `singer-sdk>=0.45.0` and will have to be replaced by a custom message reader class: https://sdk.meltano.com/en/v0.45.0/classes/singer_sdk.Target.html#singer_sdk.Target.message_reader_class